RadioItem/tests/RadioItem-specs.js

```diff
@@ -24,10 +39,76 @@ describe('RadioItem', () => {
 			</RadioItem>
 		);
 
-		const expected = 1;
-		const actual = radioItem.find(`.${css.selected}`).hostNodes().length;
+		const expected = 'selected';
+		const actual = radioItem.find(`div.${css.selected}`).prop('className');
```

Comment:
I think this is a truism :)
a better expression would be radioItem.prop('className').contains('selected')